decisions/SpanishNation.txt

```diff
@@ -217,19 +217,19 @@ country_decisions = {
 					}
```

Comment:
You may want to remove the AI conditionals at line 197, 199 etc.

Reply:
ai = yes means that AI will ignore province limit. Since he can't really avoid it. 
ME added a lots of free claims for AI to expand in. And Aragon is not blocked (like it's vanila) from going exploration expansion.